src/query.rs

```diff
@@ -77,8 +75,13 @@ fn main() {
     utils::assert_status_ok(&mut resp);
 
     let stdout = io::stdout();
-    let mut handle = stdout.lock();
-    beautician::prettify(&mut resp, &mut handle)
-        .ok()
-        .expect("failed to write response");
+    let handle = stdout.lock();
+
+    let mut buf = [0; 64*1024];
+    let mut reader = BufReader::new(resp);
+    let mut writer = BufWriter::new(handle);
+
+    while reader.read(&mut buf).unwrap() != 0 {
+        writer.write(&mut buf);
+    }
```

Comment:
I think you're wanting to do something like

```rust
let stdout = io::stdout();
let mut handle = stdout.lock();
io::copy(&mut res, &mut handle).ok().expect("failed to write response");
```

And you could wrap the resp in a BufReader if you wanted!